requirements.txt

```diff
@@ -20,7 +20,7 @@ contextily
 descartes
 scikit-plot
 tobler>=0.4
-git+ssh://git@github.com/sjsrey/spopt.git@eef58e64429ff4bc25ed2bf1d715b4ed3cc284ef
+spopt>=0.1.1
```

Comment:
we'll also need to update pin in environment.yml and .ci/ . Tests will still fail for a couple hours until anaconda rebuilds spopt